rs/types/types/src/consensus/dkg.rs

```diff
@@ -375,12 +380,32 @@ fn build_tagged_transcripts_map(
                 .ok_or_else(|| ProxyDecodeError::MissingField("TaggedNiDkgTranscript::transcript"))
                 .and_then(|t| {
                     Ok((
-                        NiDkgTag::try_from(tagged_transcript.tag).map_err(|e| {
-                            ProxyDecodeError::Other(format!(
-                                "Failed to convert NiDkgTag of transcript: {:?}",
-                                e
-                            ))
-                        })?,
+                        match tagged_transcript.tag {
+                            1 => Ok(NiDkgTag::LowThreshold),
+                            2 => Ok(NiDkgTag::HighThreshold),
```

Comment:
Same here

Reply:
Done: https://github.com/dfinity/ic/pull/2445/commits/ad29f17a6478377dd476820306547dc02490a072

Reply:
Resolving this comment, because it is addressed (see above). Let me know if you have further comments.